src/test/java/com/jcabi/github/mock/MkIssueTest.java

```diff
@@ -253,6 +255,53 @@ public void assignsUser() throws Exception {
     }
 
     /**
+     * MkIssue creates a closed event when closing an issue.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void closingGeneratesEvent() throws Exception {
+        final Issue.Smart issue = new Issue.Smart(this.issue());
+        issue.close();
+        MatcherAssert.assertThat(
+            issue.events(),
+            Matchers.<Event>iterableWithSize(1)
+        );
+        final Event.Smart closed = new Event.Smart(
+            issue.events().iterator().next()
+        );
+        MatcherAssert.assertThat(
+            closed.type(),
+            Matchers.equalTo(Event.CLOSED)
+        );
+    }
+
+    /**
+     * MkIssue creates a reopened event when closing an issue.
```

Comment:
@cvrebert documentation for a test should start with "MkIssue can create..." in this case.